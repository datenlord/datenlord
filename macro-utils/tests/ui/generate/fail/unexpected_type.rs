use macro_utils::*;

fn main() {
    let _ = generate_bytes! {
        BE;
        usize: 1024,
        u64: 65,
        u64: 20,
        u64: 507,
        u32: 0x20,
        str: b"foo.bar\0",
    };
}