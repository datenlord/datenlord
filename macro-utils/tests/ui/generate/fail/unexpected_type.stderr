error: expected one of: `u8`, `i8`, `u16`, `i16`, `u32`, `i32`, `u64`, `i64`, `str`
 --> tests/ui/generate/fail/unexpected_type.rs:6:9
  |
6 |         usize: 1024,
  |         ^^^^^
